samples/sqltransport/multi-instance/sample.md

```diff
@@ -63,11 +63,11 @@ It receives `ClientOrder` message sent by Sender and replies to them with `Clien
 snippet:Reply
 
 
-### EndpointConnectionLookup project
+### Multi-instance connection lookup
 
-The EndpointConnectionLookup provides an open `SqlConnection` for address requested either by Sender or Receiver. Note that in Version 2 it provides connections for remote endpoints only, in Version 3 it provides connections for all endpoints.
+Both sender and receiver provide a custom lookup mechanism for providing connection information for given destination. The following snippet shows lookup logic used by Sender.
```

Comment:
@indualagarsamy That is a good question :). Here is what I know:
 * There is not difference in SqlServer transport behavior. The callback in both v2 and v3 will be called with addresses of local queues (main queue + any satellite), error/audit or remote queues
 * I think the purpose of this comment was that there was a difference in callback implementation in sample for v2 and v3 - which TBH was misleading

In other words I think we should remove to comment. It brings more confusion than value.